stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_attachments_only.xml

```diff
@@ -19,10 +20,25 @@
         android:layout_height="wrap_content"
         android:padding="1dp"
         app:layout_constraintEnd_toEndOf="@+id/marginEnd"
-        app:layout_constraintStart_toStartOf="@+id/marginStart"
+        app:layout_constraintStart_toEndOf="@+id/avatarView"
         app:layout_constraintTop_toBottomOf="@id/gapView"
         />
 
+    <io.getstream.chat.android.ui.avatar.AvatarView
+        android:id="@+id/avatarView"
+        android:layout_width="@dimen/stream_ui_avatar_size_medium"
+        android:layout_height="@dimen/stream_ui_avatar_size_medium"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toStartOf="@id/marginStart"
+        app:streamAvatarHeight="@dimen/stream_ui_avatar_size_medium"
+        app:streamAvatarWidth="@dimen/stream_ui_avatar_size_medium"
```

Comment:
These attributes shouldn't be needed, we don't use these in the new AvatarView implementation at all (we just rely on the View sizing instead, see https://github.com/GetStream/stream-chat-android/pull/908/)